
package Sirikata.Network.Protocol._PBJ_Internal;
message TimeSync {
	optional fixed64 client_time = 9 ;
	optional fixed64 server_time = 10 ;
	optional uint64 sync_round = 11 ;
	enum ReturnOptions {
		REPLY_RELIABLE = 1;
		REPLY_ORDERED = 2;
	}
	optional uint32 return_options = 14 ;
	optional fixed64 round_trip = 2561 ;
}
